sysl2/sysl/sysl_test.go

```diff
@@ -599,3 +601,19 @@ func TestMain2WithDataMultipleRelationships(t *testing.T) {
 		fs, logger, main3)
 	testutil.AssertFsHasExactly(t, memFs, "/Relational-Model.png", "/Object-Model.png")
 }
+
+func TestShowBinaryInfo(t *testing.T) {
+	fakeExit := func(int) {
+		panic("os.Exit called")
+	}
+	patch := monkey.Patch(os.Exit, fakeExit)
```

Comment:
This is why hacks like calling `os.Exit` from an option handler are not a good idea. :-)